0006-cmx2cdp-protocol/README.md

```diff
@@ -105,9 +105,9 @@ _"config-response"_ message structure:
 - `contentType` (string, required) - content type of configuration, e.g.: json, protobuf.
 - `configId` (string, required) - unique identifier of endpoint configuration.
 - `statusCode` (number, required) - status code that holds meaningful information about the result of inbound message processing.
-- `reasonPhrase` (string, optional) - status code that holds meaningful information about the result of inbound message processing.
+- `reasonPhrase` (string, required) - status code that holds meaningful information about the result of inbound message processing.
 - `content` (byte[], optional) - content with configuration data.
-
+- `isApplied` (boolean, required) - shows if client has latest configuration (true if he does).
```

Comment:
What if at some point we decide to support differential config updates? In this case, the server will have to know both the current configuration and the last applied one (which would be possible to look up based on the `lastAppliedConfigId`).

With the config pull, the `lastAppliedConfigId` can be received from the client request. However, with the config push, this information must already be present on the server.

On the other hand, for safety reasons, differential config update will also require specifying the base `configId` to apply the diff to. This will require the config push message schema change. Thus, we can leave the suggested implementation for simplicity - until we design the differential config update.

@vtkhir Please approve the PR and I'll merge.